backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/humanaethica/report/ReportController.java

```diff
@@ -0,0 +1,42 @@
+package pt.ulisboa.tecnico.socialsoftware.humanaethica.report;
+
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+
+import pt.ulisboa.tecnico.socialsoftware.humanaethica.auth.domain.AuthUser;
+import pt.ulisboa.tecnico.socialsoftware.humanaethica.report.ReportService;
+import pt.ulisboa.tecnico.socialsoftware.humanaethica.report.dto.ReportDto;
+import pt.ulisboa.tecnico.socialsoftware.humanaethica.report.dto.ReportDto;
+
+import java.security.Principal;
+import java.util.List;
+
+@RestController()
+@RequestMapping
+public class ReportController {
+    @Autowired
+    ReportService reportService;
+
+    @GetMapping("/reports/administrator")
+    @PreAuthorize("(hasRole('ROLE_ADMIN'))")
+    public List<ReportDto> getAdministratorReports(Principal principal) {
```

Comment:
It make no sense, it is getting his own reports, which is none, since the admin cannot report